tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -20,61 +20,63 @@ export const feature_mcolor3: Feature<string> = {
 
 export const flavor_text: Feature<string> = {
   name: 'Flavor Text',
-  description: 'Describe your character!',
+  description:
+    "Appears when your character is examined (but only if they're identifiable - try a gas mask).",
   component: FeatureTextInput,
 };
 
 export const silicon_flavor_text: Feature<string> = {
   name: 'Flavor Text (Silicon)',
-  description: 'Describe your cyborg/AI shell!',
+  description: "Only appears if you're playing as a borg/AI.",
   component: FeatureTextInput,
 };
 
 export const ooc_notes: Feature<string> = {
   name: 'OOC Notes',
-  description: 'Talk about your character OOCly!',
+  description:
+    'Primarily for things like ERP, general, antag/whatever preferences.',
   component: FeatureTextInput,
 };
 
 export const custom_species: Feature<string> = {
   name: 'Custom Species Name',
-  description:
-    "Want to have a fancy species name? Put it here, or leave it blank if you want to use your species' default name.",
   component: FeatureShortTextInput,
 };
 
 export const custom_species_lore: Feature<string> = {
   name: 'Custom Species Lore',
-  description:
-    "Add some lore for your species! Won't show up if there's no custom species.",
+  description: "Won't show up if there's no custom species.",
   component: FeatureTextInput,
 };
-// SKYRAT EDIT ADDITION BEGIN -- RECORDS REJUVINATION
 export const general_record: Feature<string> = {
   name: 'Records - General',
-  description:
-    'Your general records! These are records that are for general viewing-things like employment, qualifications, etc. By default, anyone with a HUD/records access can view these.',
+  description: 'Viewable by anyone with any records access.',
```

Comment:
Why was this shortened?

Reply:
We dont have actual regulations on what goes in records, so I thought it was irrelevant, though a guide on what COULD be in here could be a good idea I suppose

Reply:
Better to have then to have not

Reply:
i changed my mind overnight, ill put back the context